extensions/dig/package.json

```
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "dig",
+  "title": "Dig (DNS Lookup)",
+  "description": "DNS lookup and nameserver query using the built-in host command",
+  "icon": "command-icon.png",
+  "author": "danielbahl",
+  "categories": [
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "dig",
+      "subtitle": "Query Hostname",
```

I would actually switch those two
```suggestion
      "title": "Query Hostname",
      "subtitle": "dig",
```